src/examples/inline-videos/render-inline-video-example.js

```diff
@@ -28,7 +28,7 @@ function start(params) {
     `
 
     const videoExampleEl = el.querySelector('.rb-example__video');
-    const exampleVideoEndpoint = '/v3/api/composition/v3/query/en-INT?filter[type]=videos&page[limit]=1';
+    const exampleVideoEndpoint = '/v3/api/graphql/v1/v3/query/en-INT?filter[type]=videos&page[limit]=1&rb3Schema=v1:video';
```

Comment:
In this case it would be best to even remove the prefix, as it's added in the consumer app

```suggestion
    const exampleVideoEndpoint = '/v3/query/en-INT?filter[type]=videos&page[limit]=1';
```